### ISSUE TITLE
UI bug - beta FW

### ISSUE THREAD

**Issue by rustysun9** [BruceDevices/firmware#567]
**Describe the bug:**

When selecting RF - Custom SubGhz - SD card - BruceRF folder, you only get popup menu for creating new folder and other unrelated to RF menu items. After selecting Main Menu to return, black screen is displayed with empty battery icon on top right side, After this, moving through UI is slow and unresponsive.

**Steps To Reproduce:**

Select RF - Custom SubGhz - SD card - BruceRF folder - Main Menu

- Device used: LilyGO T Embed cc1101 on 96955c4 beta firmware.
Official release 1.8 is OK. 

**Expected behavior:**

Select .sub file saved on SD card for replay data using CC1101.

**Actual behavior:**

Unresponsive UI.

**Additional context / logs:**

Logs from serial:
```
[  8256][V][esp32-hal-ledc.c:231] analogWrite(): GPIO 21 - Using Channel 7, Value = 255
{
  "priColor": "ffff",
  "secColor": "dfff",
  "bgColor": "0",
  "rot": 3,
  "dimmerSet": 10,
  "bright": 100,
  "tmz": 0,
  "soundEnabled": 1,
  "wifiAtStartup": 0,
  "webUI": {
    "user": "admin",
    "pwd": "bruce"
  },
  "wifiAp": {
    "ssid": "BruceNet",
    "pwd": "brucenet"
  },
  "wifi": {},
  "irTx": 2,
  "irRx": 1,
  "rfTx": 8,
  "rfRx": 18,
  "rfModule": 0,
  "rfFreq": 433.92,
  "rfFxdFreq": 1,
  "rfScanRange": 3,
  "rfidModule": 1,
  "gpsBaudrate": 9600,
  "startupApp": "",
  "wigleBasicToken": "",
  "devMode": 0,
  "disabledMenus": [],
  "qrCodes": [
    {
      "menuName": "Bruce AP",
      "content": "WIFI:T:WPA;S:BruceNet;P:brucenet;;"
    },
    {
      "menuName": "Bruce Wiki",
      "content": "https://github.com/pr3y/Bruce/wiki"
    },
    {
      "menuName": "Bruce Site",
      "content": "https://bruce.computer"
    },
    {
      "menuName": "Rickroll",
      "content": "https://youtu.be/dQw4w9WgXcQ"
    }
  ]
}[  8305][I][config.cpp:181] saveFile(): config file written successfully
[  8332][W][LittleFS.cpp:77] begin(): LittleFS Already Mounted!
[  8406][I][config.cpp:162] fromFile(): Using config from file
unsupported serial command: [   708][v][esp32-t
failed: [   708][V][esp32-t
$ unsupported serial command: "
failed:   "
$ Selected: Spectrum
[ 36644][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[ 38653][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[ 40673][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[ 42699][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[ 44707][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[ 46737][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[ 48745][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
[ 50768][E][Wire.cpp:513] requestFrom(): i2cRead returned Error 263
```

**Comment by bmorcelli** [comment]
I tested here many times and couldn't reproduce it..

can you send the .sub file you are using? maybe it is overflowing some buffer and affecting I2C addresses

**Comment by rustysun9** [comment]
Hi Pirata, I couldnt even open BruceRF folder to select .sub file. All sub files were on SD card inside this folder and after selecting directory, only some menu poped up.
Anyhow, I just flashed latest beta 7ba77ca on two different T-embed cc1101 devices and everything is ok now. Not sure what was the problem yesterday. Now when you mentioned I2C....is it possible that problem was because I did not choose cc1101 in config after flashing? I think I didnt do it on previous beta. Although I didnt do it also after flashing official release on second Tembed. 
Will report in case something simmilar happens again and will try to do some more investigation on my side. It was already late yesterday.
